### PR TITLE
add detailed information in error logs

### PR DESCRIPTION
**Pull request by gregwebs**
When going through the TiKV startup log I saw messages that were difficult to understand. These changes would help me understand the log messages (and code in one case) more quickly.

## What are the type of the changes? (mandatory)

This change affects log messages, but I don't think we have any contract for those.

## How has this PR been tested? (mandatory)

`make`. I have not actually looked at the log output after these changes.

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

no

## Does this PR affect tidb-ansible update? (mandatory)

no

### CODE REVIEW

**Comment by Hoverbear** [comment]
@gregwebs Can you run `make format`  and commit the changes?

File: src/util/sys/mod.rs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pub const HIGH_PRI: i32 = -1;
 
 #[cfg(target_os = "linux")]
-pub mod pri {
+pub mod linux_thread {
```
Comment (siddontang) [inline src/util/sys/mod.rs]: seem using `thread` for the mod name is enough.

File: src/util/sys/mod.rs [hunk 2]
```diff
@@ -60,7 +60,7 @@ pub mod pri {
 }
 
 #[cfg(not(target_os = "linux"))]
-pub mod pri {
+pub mod process_thread {
```
Comment (overvenus) [inline src/util/sys/mod.rs]: It should be `thread` too, otherwise it wont compile on platforms other than `linux`.

Comment (siddontang) [reply src/util/sys/mod.rs]: +1

File: src/util/config.rs [hunk 3]
```diff
@@ -715,6 +715,8 @@ mod check_data_dir {
             static ref GETMNTENT_LOCK: Mutex<()> = Mutex::new(());
         }
 
+        let op = "data-dir.fsinfo.get";
```
Comment (siddontang) [inline src/util/config.rs]: can we move the op to the error directly?

Comment (siddontang) [reply src/util/config.rs]: seem only one error use it

File: src/util/config.rs [hunk 4]
```diff
@@ -740,15 +742,16 @@ mod check_data_dir {
...
+                    "{}: path: {:?} not find in mountable",
+                    op, path
                 )));
             }
             Ok(fs)
         }
     }
 
     fn get_rotational_info(fsname: &str) -> Result<String, ConfigError> {
+        let op = "data-dir.rotation.get";
```
Comment (siddontang) [inline src/util/config.rs]: seem rotation is not related to data-dir?

File: src/util/config.rs (hunk 3 above)
Comment (gregwebs) [reply src/util/config.rs]: At the moment that may be the case, but a new error may be added. Also, this makes for consistency with other functions.

File: src/util/config.rs (hunk 4 above)
Comment (gregwebs) [reply src/util/config.rs]: It is only called by check_data_dir right now, so this matches the code. I agree that this relationship is not guaranteed in code and it is possible for this to get out of sync.

**Comment by siddontang** [comment]
please fix the conflicts and signed off your commits 

@gregwebs

**Comment by siddontang** [comment]
Please approve it @overvenus

**Comment by gregwebs** [comment]
Lost my approvals from the rebase. @siddontang @zhangjinpeng1987

**Pull request merged**